external_api/external_api_utilities.py

```diff
@@ -0,0 +1,36 @@
+def structure_es_result(result):
```

Comment:
Try removing this file all together if possible. It creates window for inconsistent communication amongst services and extra parsing etc. There is no strong utility requirement yet. Either move code in it's respective parent function or modify parent function to eliminate necessity of different data structures.

Reply:
Done